test/integration/testArrayBind.js

```diff
@@ -440,34 +461,42 @@ describe('Test Array Bind', function ()
   });
 });
 
-describe('testArrayBind - full path', function ()
+describe('Test Array Bind - full path', function ()
 {
+  const DATABASE_NAME = connOption.valid.database;
+  const SCHEMA_NAME = connOption.valid.schema;
+
   this.timeout(600000);
   var connection;
-  var createABTable = `create or replace table  ${DATABASE_NAME}.${SCHEMA_NAME}.testAB(colA string, colB number, colC date, colD time, colE TIMESTAMP_NTZ, colF TIMESTAMP_TZ)`;
-  var insertAB = `insert into  ${DATABASE_NAME}.${SCHEMA_NAME}.testAB values(?, ?, ?, ?, ?, ?)`;
+  const fullTableName = `${DATABASE_NAME}.${SCHEMA_NAME}.testAB`;
+  var createABTable = `create or replace table ${fullTableName}(colA string, colB number, colC date, colD time, colE TIMESTAMP_NTZ, colF TIMESTAMP_TZ)`;
+  var insertAB = `insert into ${fullTableName} values(?, ?, ?, ?, ?, ?)`;
   
   before(function (done)
   {
     connection = snowflake.createConnection({
-      accessUrl: connOption.valid.accessUrl,
-      account: connOption.valid.account,
-      username: connOption.valid.username,
-      password: connOption.valid.password,
-      warehouse: connOption.valid.warehouse,
-      role: connOption.valid.role,
+      ...connOption.valid,
+      // Set schema and database to null to ensure that full path to table is passed in commands
+      schema: undefined,
+      database: undefined,
       arrayBindingThreshold: 3
     });
-    testUtil.connect(connection, function ()
+    testUtil.connect(connection, function (err)
```

Comment:
Maybe it will be better to replace that before to async and await to connect and create table?